packages/http-client-csharp/emitter/test/Unit/dotnet-sdk-validation.test.ts

```diff
@@ -0,0 +1,54 @@
+import { Program } from "@typespec/compiler";
+import { TestHost } from "@typespec/compiler/testing";
+import { strictEqual } from "assert";
+import { beforeEach, describe, expect, it } from "vitest";
+import { validateDotNetSdkVersion } from "../../src/emitter.js";
+import { createEmitterTestHost, typeSpecCompile } from "./utils/test-util.js";
+
+describe("Test validateDotNetSdkVersion", () => {
+  let runner: TestHost;
+  let program: Program;
+  const minVersion = 8;
+
+  beforeEach(async () => {
+    runner = await createEmitterTestHost();
+    program = await typeSpecCompile(
+      `
+            op test(
+                @query
+                @encode(DurationKnownEncoding.ISO8601)
+                input: duration
+              ): NoContentResponse;
+      `,
+      runner,
+    );
+  });
+
+  it("should return true for installed SDK version whose major equals min supported version", async () => {
```

Comment:
We should have a case for when the SDK is not present on the system.   (no `dotnet` command)

Reply:
@jsquire I added the test for `no dotnet`. Please take a look. thanks